framework/src/binary_manager/binary_manager_notify.c

```diff
@@ -27,7 +27,7 @@
 #include <binary_manager/binary_manager.h>
 #include "binary_manager_internal.h"
 
-int binary_manager_notify_binary_started(void)
+binmgr_result_type_e binary_manager_notify_binary_started(void)
```

Comment:
In this function, `binary_manager_xxx` APIs return `binmgr_result_type_e`, not int. But you declared `int ret` for getting return of them. Please check this.

Reply:
```
binmgr_result_type_e ret;
ret = binary_manager_set_request(&request_msg, BINMGR_NOTIFY_STARTED, BINMGR_REQUEST_MQ);
if (ret != BINMGR_OK) {
   return ret;
}
```